src/Agent/NewRelic/Profiler/Configuration/Configuration.h

```diff
@@ -680,6 +680,20 @@ namespace NewRelic { namespace Profiler { namespace Configuration {
                 return 0;
             }
 
+            // look for -functions-worker-id (or --worker-id, to instrument during local testing) in the command line and instrument if found
```

Comment:
Are these documented? Are there also "short" versions of these flags we should check for?

Reply:
I don't think any of it is documented -- this is all based on observed behavior.

Reply:
Works for me!